### PR TITLE
Add attribute reference to google_service_networking_connection

### PR DESCRIPTION
**Pull request by carlpett**
The `google_service_networking_connection` resource doesn't have an Attributes reference section. The resource currently exposes a `peering` field that should be documented.

If this PR is for Terraform, I acknowledge that I have:

- [X] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/tests) (for handwritten resources or update tests).
- [ ] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [x] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

```release-note:none
```

### CODE REVIEW

File: mmv1/third_party/terraform/website/docs/r/service_networking_connection.html.markdown [hunk 1]
```diff
@@ -50,6 +50,13 @@ The following arguments are supported:
   this service provider. Note that invoking this method with a different range when connection
   is already established will not reallocate already provisioned service producer subnetworks.
 
+## Attributes Reference
+
+In addition to the arguments listed above, the following computed attributes are exported:
+
+    peering - (Computed) The name of the created peering.
```
Comment (c2thorn) [inline mmv1/third_party/terraform/website/docs/r/service_networking_connection.html.markdown:57]:
```suggestion
* `peering` - (Computed) The name of the VPC Network Peering connection that was created by the service producer.
```

**Review by c2thorn — CHANGES_REQUESTED** [review]
Thanks for getting started on this! I've copied the description of the field from the API exactly, we usually do something like that.

**Review by c2thorn — APPROVED** [review]

File: mmv1/third_party/terraform/website/docs/r/service_networking_connection.html.markdown (hunk 1 above)
Comment (carlpett) [reply mmv1/third_party/terraform/website/docs/r/service_networking_connection.html.markdown]: Thanks! :+1:

**Pull request merged**